seqr/utils/elasticsearch/es_search.py

```diff
@@ -43,7 +45,7 @@ def __init__(self, families, previous_search_results=None, skip_unaffected_famil
 
         self._skipped_sample_count = defaultdict(int)
         if skip_unaffected_families:
-            for index, family_samples in self.samples_by_family_index.items():
+            for index, family_samples in list(self.samples_by_family_index.items()):
```

Comment:
It should be okay without adding `list()`. But, Python 3 reports `dictionary changed size during iteration` error. After adding `list()`, the error disappeared.